install/update.php

```diff
@@ -37,9 +37,6 @@
 include_once (GLPI_ROOT . "/inc/based_config.php");
 include_once (GLPI_ROOT . "/inc/db.function.php");
 
-$translation_cache = Config::getCache('cache_trans');
```

Comment:
Translations cache must be reset on update

Reply:
Right. I thought that cache was automatically update on local files change, but I looked at Translator code and I did not see such mechanism.

Reply:
Currently, to avoid emtpy directories that are not handled code side, usage seems to rely on a `remove.txt` file.
Event if the code checks for the directory presence, it's certainly not a problem if the directory do exists from checkout.

Reply:
NP, I remember we had to clear at this point because of some issues ;)